lib/bootbasetest.pm

```diff
@@ -10,19 +10,17 @@ sub post_fail_hook {
         record_info 'Seems that the display manager failed';
     }
 
-    # if we found a shell, we do not need the memory dump
-    if (!(match_has_tag('emergency-shell') or match_has_tag('emergency-mode'))) {
-        die "save_memory_dump not implemented, no way to save memory_dump" unless check_var('BACKEND', 'qemu');
-        select_console 'root-console';
-        diag 'Save memory dump to debug bootup problems, e.g. for bsc#1005313';
-        save_memory_dump;
-    }
-
     # crosscheck for text login on tty1
     select_console 'root-console';
 
     # collect and upload some stuff
-    export_logs();
+    opensusebasetest->export_logs();
```

Comment:
please use `$self->SUPER::export_logs` instead. Can you explain in your commit message and PR description why this seems to be necessary now? I doubt this *never* worked.

Reply:
As explain here: https://progress.opensuse.org/issues/53192 
Because:
```
post_fail_hook failed: Undefined subroutine &bootbasetest::export_logs called at /var/lib/openqa/share/tests/opensuse/lib/bootbasetest.pm line 25.
```

Reply:
I see. So I assume this broke when the class "bootbasetest" was introduced then